core/jobs/batch_jobs/audit_rule_validation_jobs.py

```diff
@@ -613,7 +586,7 @@ def _filter_invalid_numeric_ans_group(
                         rule_spec_dict: state_domain.RuleSpecDict = {
                             'rule_type': 'Equals',
                             'inputs': {
-                                'x': int(rule_value_x)
+                                'x': rule_spec.inputs['x']
```

Comment:
Is `rule_specs.inputs['x']` stored as a float already? If so, then this is fine. Otherwise, we'd need to convert to float

Reply:
It can be a decimal value or an int value that is why I'm just replacing it with the value that I receive without any conversion.